consensus-merge/1.0.0/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:20.04
+LABEL maintainer="Nathan Johnson <johnsonn13@chop.edu>"
+LABEL description="This is a script to create a single merged VCF from multiple single-caller somatic VCFs"
+
+RUN apt update -y && apt install -y curl
+
+# Download and install conda into /usr/bin
+ENV MINICONDA_VERSION=py38_4.9.2
+RUN curl -sL https://repo.anaconda.com/miniconda/Miniconda3-${MINICONDA_VERSION}-Linux-x86_64.sh -o /tmp/miniconda.sh && \
```

Comment:
Please consolidate, as much as possible (in this case, you can do all) the `RUN` commands as a single one, and string with `&&`. Doing so cuts the image size, in this case in half, as each `RUN` call adds a layer to the docker image and sometimes significantly inflate the size.